src/app/Layouts/defaultPageWrapper.jsx

```diff
@@ -1,13 +1,27 @@
 import React, { useContext } from 'react';
 import { node } from 'prop-types';
 import GlobalStyles from '@bbc/psammead-styles/global-styles';
+import styled from 'styled-components';
+import { C_GHOST } from '@bbc/psammead-styles/colours';
 import HeaderContainer from '../containers/Header';
 import FooterContainer from '../containers/Footer';
 import ManifestContainer from '../containers/Manifest';
 import ServiceWorkerContainer from '../containers/ServiceWorker';
 import { ServiceContext } from '../contexts/ServiceContext';
 import WebVitals from '#app/containers/WebVitals';
 
+const Wrapper = styled.div`
+  min-height: 100vh;
```

Comment:
Do we already use `vh`? Doesn't look like Opera Mini supports it

Reply:
oh I didn't realise that. 100% will still do the job in this case then